src/api/VoteMonitor.Api.Location/Services/XslxFileLoader.cs

```diff
@@ -12,10 +13,26 @@ namespace VoteMonitor.Api.Location.Services
     public class XlsxFileLoader : IFileLoader
     {
         private static readonly string[] SUFFIXES = { ".xlsx", ".xls" };
-        private static readonly int COD_JUDET_INDEX = 1;
-        private static readonly int ADRESA_INDEX = 7;
-        private static readonly int SIRUTA_INDEX = 3;
-        private static readonly int NR_SECTIE_INDEX = 5;
+        private static PollingStationExcelHeader header = new PollingStationExcelHeader();
+
+        public byte[] ExportHeaderInformation()
+        {
+            using (var excelPackage = new ExcelPackage())
+            {
+                var workSheet = excelPackage.Workbook.Worksheets.Add("HeaderInfo");
+
+                var maximumColumnIndex = header.Columns.OrderByDescending(x => x.Index).FirstOrDefault().Index;
```

Comment:
Can we use Max(column => column.Index) instead of OrderByDescending().FirstOrDefault().Index ?
If we chose the Max method, the evaluation will be faster(even tho it's insignificant at this amount of data) and more readable.

Reply:
Yes, good idea! Did the change. Thank you!